src/app/modules/main/activity_center/module.nim

```diff
@@ -78,7 +78,10 @@ method convertToItems*(
 
       let chatDetails = self.controller.getChatDetails(n.chatId)
       # default section id is `Chat` section
-      let sectionId = if(chatDetails.communityId.len > 0): chatDetails.communityId else: conf.CHAT_SECTION_ID
+      let sectionId = if(chatDetails.communityId.len > 0):
```

Comment:
@jrainville please add 2 spaces more to the lines 82, 83 and 84 cause `if` belongs to the inner scope here.